compiler_gym/envs/compiler_env.py

```diff
@@ -88,6 +88,16 @@ def close(self):
         """
         raise NotImplementedError("abstract method")
 
+    @property
+    @abstractmethod
+    def timeout(self) -> float:
+        raise NotImplementedError("abstract method")
+
+    @timeout.setter
+    @abstractmethod
+    def timeout(self, timeout: Optional[float]):
+        raise NotImplementedError("abstract method")
+
```

Comment:
Is this property needed? I don't think it's used anywhere, and `timeout` arg is added as a param to the relevant methods.

Reply:
Done!